packages/core/components/Filters.tsx

```diff
@@ -378,8 +378,12 @@ const Filters = (props: FilterProps) => {
       const { active, queuedActive, label, filterStyle } = singleFilter as VizFilter
 
       handleSorting(singleFilter)
+      singleFilter.orderedValues =
+        singleFilter.orderedValues === undefined || singleFilter.orderedValues.length === 0
+          ? singleFilter.values
+          : singleFilter.orderedValues
 
-      singleFilter.values?.forEach((filterOption, index) => {
+      singleFilter.orderedValues?.forEach((filterOption, index) => {
```

Comment:
I don't think this is necessary. values should be accurate. if there are orderedValues then handleSorting should have fixed that.